proto/vtctldata.proto

```diff
@@ -414,6 +414,14 @@ message GetVSchemaResponse {
   vschema.Keyspace v_schema = 1;
 }
 
+message ApplyVSchemaRequest {
+  string keyspace = 1;
+  vschema.Keyspace v_schema = 2;
+}
+
+message ApplyVSchemaResponse {
```

Comment:
I wonder if it will be useful to include the final, saved vschema here (after reading it back out of the topo, similar to how [`ChangeTabletType`](https://github.com/vitessio/vitess/blob/master/go/vt/vtctl/grpcvtctldserver/server.go#L110-L123) works

Reply:
and similar to how the current vtctlclient command works

Reply:
done